README.md

```diff
@@ -1,24 +1,163 @@
+The directory name is the name of the project. Each directory contains the following files:
+
+#### Raw intermediate dataset files
+> [!IMPORTANT]
+> `YYMMDD-batchName` directory 
+ 
+These sub-directories contains raw output files from the manual annotation process created by the annotation tool (or by hand like a `.csv` file). 
+Different annotation tools create different file formats with diverse formatting. The `YYMMDD-` prefix _must_ indicate the time when a batch of annotation is conducted. (e.g., when the batch is decided to be annotated)
+These prefixes are important for easy sorting of annotation processes and machine ingestion of the raw data. 
+The `batchName` part of the directory name _must_ match only one of `.txt` files in the batches. 
+
+#### Codebase for conversion
+> [!IMPORTANT]
+> (usually) `process.{sh,py}` and dependencies
+ 
+A piece of software to process the raw manual annotations and generate the publicly-available "gold" dataset. 
+In addition to the main code file, if the code requires additional dependencies/scripts, they can stay in the same directory. 
+The dependencies information can be written down in in the `README.md` (below) file or in a machine-friendly file with the list of dependencies (e.g. `requirements.txt` for `pip`).
+
+#### Final "Gold" dataset files
+> [!IMPORTANT]
+> `golds` directory
+
+This directory contains the public "gold" dataset generated by the above script. The script _must_ generate one file per GUID (video/audio/text document) and the number of gold files in this directory _must_ match the sum of GUIDs in all batches (`YYMMDD-xxx` subdirectories) annotated. 
+The gold dataset is a set of files that are in a format that is ready for use with the newly developed tools. I.e. The raw file must be properly formatted so that tools in the next step of the process can use this dataset.
+
+#### Information README
+> [!IMPORTANT] 
+> `README.md` (and possibly `guidelines.md`)
+
+Project-specific information, including but not limited to:
+* Annotation project name
+* Annotator summary: Some basic demographic information about the annotators. Age group, native languages, language proficiency, occupational characteristics, etc.
+* Annotation environment/tool information (name, version, link, tool used, user manual, etc.)
+* Project changes: version changes, addition of new batches, change in annotator personnel, etc.
+* Raw-to-gold conversion code explanation 
+  * dependencies, short description of `process.py`
+  * file formats of raw and gold
+  * field description, _datatype_
+  * differences, added information, discarded information during `process.py`, added info during process.py, etc.)  
+* Annotation guidelines - sometimes this is a separate file: `guidelines.md`: How to annotate in this project, aka scheme. This section should give sufficient information for the replication of the annotation to produce almost exact similar raw datasets.
```

Comment:
(moving my previous comment on one shared google docs) 

---
Few more things to consider w.r.t usage of term _ replication_. 

* Do we want to do annotation verification (meaning, how can we trust annotation from one particular annotator?)
* Do we want to do cross-validation of annotation with multiple annotators working on the same data? (meaning, are we willing to pay triple+ wage and triple+ time for additional annotators, additional adjudicator/curator, additional adjudication tool development?)
* At some point, do we plan to conduct crowdsourced anntoations? (I'm very skeptical on this most due to copyright issues with AAPB assets, but we can use crowdsource and non-AAPB (free) data to create larger, faster and cheaper datasets)
* Final remark; having a wonderfully written guidelines and having annotators follow exactly what's in the guidelines are completely different stories. So in all cases, guidelines is not sufficient to provide any reproducibility of the annotation data. At best, it can provide a measurement of the "reliability" of single-pass of annotation, only if annotation is done with duplicate effort (multi-annotator on the same data) and agreement is measured. 

My conclusion; I think we should not use any mention of "reproducibility" or "reliability" or "annotation quality"  in this document until we have a better idea how we measure those indices.

Reply:
My general perspective on guidelines:  The guidelines/conventions say what we want from annotators.  However, we do, of course, expect that humans will always fail to follow guidelines and instructions perfect.

My general perspective on verification and/or validation of data:  I could see why that could _possibly_ become desirable for certain evaluations.  However, I'd recommend not worrying too much about it now.  To me, the most important thing is to have good documentation of the datasets, so that we can make good assessments about whether they are suitable for a given purpose (and not misused beyond their appropriate scope).

Reply:
I think this is well discussed, but no changes are evidently required. Shall we resolve this conversation?

Reply:
How about removing "replication of the annotation", and repharasing the sentence?

Reply:
How is this? 
"This section should give sufficient documentation for how the annotation was done and the conditions/assumptions under which the dataset exists."

Reply:
this was added.